client/reader/full-post/main.jsx

```diff
@@ -346,7 +346,7 @@ FullPostDialog = React.createClass( {
 			}
 
 			if ( shouldShowComments ) {
-				buttons.push( <CommentButton key="comment-button" commentCount={ this.props.commentCount } onClick={ this.handleCommentButtonClick } tagName="div" /> );
+				buttons.push( <CommentButton key="comment-button" commentCount={ this.props.commentCount } onClick={ this.handleCommentButtonClick } tagName="div" postId={ post.ID } siteId={ post.site_ID } /> );
```

Comment:
Do we need to pass `commentCount` if it will be retrieved from the store via `getPostTotalCommentsCount()`?